mws/mws.go

```diff
@@ -72,17 +72,27 @@ type Network struct {
 	GcpNetworkInfo   *GcpNetworkInfo      `json:"gcp_network_info,omitempty"`
 }
 
+// GcpVpcEndpointInfo is the objecy that configures GCP Private Service Connect endpoints.
+type GcpVpcEndpointInfo struct {
+        PscConnectionId     string `json:"psc_connection_id,omitempty" tf:"computed"`
+        ProjectId           string `json:"project_id"`
+        PscEndpointName     string `json:"psc_endpoint_name"`
+        EndpointRegion      string `json:"endpoint_region"`
+        ServiceAttachmentId string `json:"service_attachment_id,omitempty" tf:"computed"`
+}
+
 // VPCEndpoint is the object that contains all the information for registering an VPC endpoint
 type VPCEndpoint struct {
-	VPCEndpointID           string `json:"vpc_endpoint_id,omitempty" tf:"computed"`
-	AwsVPCEndpointID        string `json:"aws_vpc_endpoint_id"`
-	AccountID               string `json:"account_id,omitempty"`
-	VPCEndpointName         string `json:"vpc_endpoint_name"`
-	AwsVPCEndpointServiceID string `json:"aws_endpoint_service_id,omitempty" tf:"computed"`
-	AWSAccountID            string `json:"aws_account_id,omitempty" tf:"computed"`
-	UseCase                 string `json:"use_case,omitempty" tf:"computed"`
-	Region                  string `json:"region"`
-	State                   string `json:"state,omitempty" tf:"computed"`
+	VPCEndpointID           string              `json:"vpc_endpoint_id,omitempty" tf:"computed"`
+	AwsVPCEndpointID        string              `json:"aws_vpc_endpoint_id"`
+	AccountID               string              `json:"account_id,omitempty"`
+	VPCEndpointName         string              `json:"vpc_endpoint_name"`
+	AwsVPCEndpointServiceID string              `json:"aws_endpoint_service_id,omitempty" tf:"computed"`
+	AWSAccountID            string              `json:"aws_account_id,omitempty" tf:"computed"`
+	UseCase                 string              `json:"use_case,omitempty" tf:"computed"`
+	Region                  string              `json:"region"`
```

Comment:
this is now optional

```suggestion
	Region                  string              `json:"region,omitempty"`
```

Reply:
Done.